app/views/courses/EnrollmentsView.coffee

```diff
@@ -1,90 +1,79 @@
-app = require 'core/application'
-CreateAccountModal = require 'views/core/CreateAccountModal'
-Classroom = require 'models/Classroom'
+RootView = require 'views/core/RootView'
 Classrooms = require 'collections/Classrooms'
-CocoCollection = require 'collections/CocoCollection'
-Course = require 'models/Course'
+State = require 'models/State'
 Prepaids = require 'collections/Prepaids'
-RootView = require 'views/core/RootView'
-stripeHandler = require 'core/services/stripe'
 template = require 'templates/courses/enrollments-view'
-User = require 'models/User'
 Users = require 'collections/Users'
-utils = require 'core/utils'
-Products = require 'collections/Products'
+Courses = require 'collections/Courses'
+HowToEnrollModal = require 'views/teachers/HowToEnrollModal'
+TeachersContactModal = require 'views/teachers/TeachersContactModal'
+ActivateLicensesModal = require 'views/courses/ActivateLicensesModal'
 
 module.exports = class EnrollmentsView extends RootView
   id: 'enrollments-view'
   template: template
-  numberOfStudents: 15
-  pricePerStudent: 0
 
-  initialize: (options) ->
-    @ownedClassrooms = new Classrooms()
-    @ownedClassrooms.fetchMine({data: {project: '_id'}})
-    @supermodel.trackCollection(@ownedClassrooms)
-    @listenTo stripeHandler, 'received-token', @onStripeReceivedToken
-    @fromClassroom = utils.getQueryVariable('from-classroom')
+  events:
+    'input #students-input': 'onInputStudentsInput'
+    'click #enroll-students-btn': 'onClickEnrollStudentsButton'
+    'click #how-to-enroll-link': 'onClickHowToEnrollLink'
+    'click #contact-us-btn': 'onClickContactUsButton'
+
+  initialize: ->
+    @state = new State({
+      totalEnrolled: 0
+      totalNotEnrolled: 0
+      classroomNotEnrolledMap: {}
+      classroomEnrolledMap: {}
+      numberOfStudents: 15
+      totalCourses: 0
+      prepaidGroups: {
+        'available': []
+        'pending': []
+      }
```

Comment:
Not sure why I left the single-quotes there... but I like curly braces. Gives the document a little more form. Same with parens.